src/data/objects/account.yaml

```diff
@@ -16,10 +27,14 @@ schema:
     type: String
     description: City portion of this account's billing address.
     value: Philadelphia
-  company:
+  state:
     type: String
-    description: This account's company name.
-    value: My Company LLC
+    description: State (or province) portion of this account's billing address.
```

Comment:
Might be helpful to explain in detail that if the customer is not in the US, please enter the province here. I know you have (or province) in parens, but I know from experience dealing with a lot of Asian folks - for example - that it's not entirely clear.

That one Thai customer account from two months ago comes to mind as well.

I guess I'm saying, let's add a little more detail in the description to be 100% perfectly clear.